GetStream/stream-chat-swift
# [CIS-1888] Fix DM Channel with multiple members displaying only 1 user avatar

### 🔗 Issue Links
CIS-1888

### 🎯 Goal
Fix issue of DM Channel with multiple members and without channel image assigned showing the image of only one member instead of showing the merged avatar.

### 🛠 Implementation
Instead of using `isDirectMessageChannel` to show 1:1 avatar image for the channel, do this only if the memberCount is 2.

### 🎨 Showcase
The change can be seen in the new snapshot test added 👍 

### 🧪 Manual Testing Notes
- Open Channel List
- Create a DM Channel with multiple users
- Verify that the DM Channel has the avatar merged

### ☑️ Contributor Checklist
